docs/js/header.js

```diff
@@ -0,0 +1,42 @@
+// ヘッダー
+// モバイルメニューの切り替え機能
+document.addEventListener('DOMContentLoaded', function() {
+  const toggleButton = document.getElementById('mobile-menu-toggle');
+  const mobileMenu = document.getElementById('mobile-menu');
+  const hamburgerIcon = document.getElementById('hamburger-icon');
+  const closeIcon = document.getElementById('close-icon');
+
+    // メニューを閉じる関数
+  function closeMobileMenu() {
+    mobileMenu.classList.add('hidden');
+    hamburgerIcon.classList.remove('hidden');
+    hamburgerIcon.classList.add('block');
+    closeIcon.classList.add('hidden');
+    closeIcon.classList.remove('block');
+  }
+
+  if (toggleButton && mobileMenu && hamburgerIcon && closeIcon) {
```

Comment:
[imo]

こちら、モバイル画面で表示中であることを表している認識です。
isSpやshowMobileHeaderのような説明変数を使うと見やすいと思います。
```suggestion
const showMobileHeader = toggleButton && mobileMenu && hamburgerIcon && closeIcon
  if ( showMobileHeader ) {
```